.github/workflows/ci.yml

```diff
@@ -69,3 +68,32 @@ jobs:
     - name: Make sure `schema.graphql` is up to date
       working-directory: backend
       run: cargo run --bin export-schema | diff -u --color=always - ../frontend/src/schema.graphql
+
+    # Deployment
+    - name: Prepare binary and demo data for deployment
+      run: |
+        cp -v backend/target/release/tobira .deployment/files/
+        cp -v scripts/fixtures.sql .deployment/files/
+
+    - name: prepare deploy key
+      env:
+        DEPLOY_KEY: ${{ secrets.DEPLOY_KEY }}
+      run: |
+        install -dm 700 ~/.ssh/
+        echo "${DEPLOY_KEY}" > ~/.ssh/id_ed25519
+        chmod 600 ~/.ssh/id_ed25519
+        ssh-keyscan tobira.opencast.org >> ~/.ssh/known_hosts
+      if: ${{ github.event_name == 'push' }}
+
+    - name: install ansible postgres extensions
+      run: ansible-galaxy collection install community.general
+
+    - name: deploy tobira branch
+      working-directory: .deployment
+      run: >
+        ansible-playbook
+        --private-key=~/.ssh/id_ed25519
+        --extra-vars="branch='${GITHUB_REF#refs/heads/}'"
```

Comment:
Wait, how does that work? `GITHUB_REF` is an env variable I assume? What does `#refs/heads/` do? And is the PR number already included in here if there is a PR? Because otherwise if someone creates a PR from their `master` branch, they overwrite our deployment.

Reply:
Currently, this will _only_ run on branches, not on pull requests.

> What does #refs/heads/ do?

It removes `refs/heads/` from the beginning of the git reference so we don't get `refs/heads/master`